randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -516,7 +545,8 @@ Extra - asset_id: 14
   * Normal Door to Terminator Room/Door to Parlor
   > Door to Landing Site
       Any of the following:
-          Screw Attack or Speed Booster
+          Screw Attack
+          Speed Booster and Disabled Entrance Randomizer
           All of the following:
```

Comment:
Can you change this below with the "any bombs" template?

Reply:
Done.